run_tests

```diff
@@ -1,3 +1,13 @@
 #!/bin/bash
 
-python3 -m unittest "$@"
+if command -v python3 >/dev/null; then
+    # linux or macos
+    _python=python3
+else
+    # windows
+    _python=python
```

Comment:
How this `run_tests` is used in windows? (I mean, it runs with a `/bin/bash`????)

Reply:
you can be running bash on windows -- and in fact travis does :-)